tools/rna_tools/rnaformer/infer_rnaformer.xml

```diff
@@ -0,0 +1,198 @@
+<tool id="infer_rnaformer" name="@EXECUTABLE@" version="@TOOL_VERSION@" profile="22.05">
+    <description>Predict the secondary structure of an RNA with RNAformer</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements">
+        <requirement type="package" version="1.83">biopython</requirement>
+    </expand>
+    <command detect_errors="exit_code"><![CDATA[
+    mkdir -p './model' &&
+    wget -O './model/RNAformer_32M_state_dict_intra_family_finetuned.pth' 'https://ml.informatik.uni-freiburg.de/research-artifacts/RNAformer/models/RNAformer_32M_state_dict_intra_family_finetuned.pth'
```

Comment:
can you host that on github? that might be more stable ... same for the yaml

Reply:
I asked the authors, they will check if they can request LFS space for this.